src/models/monster.ts

```diff
@@ -27,6 +27,7 @@ export interface Monster extends Element {
 		characteristic: Characteristic;
 		value: number;
 	}[];
+	withCaptain?: string;
```

Comment:
Probably better to default to `''` than make it optional.

Reply:
I don't agree that setting a property to a default value rather than making it optional is better, but i can do that if you prefer

Reply:
Yes please.